x-pack/plugins/fleet/common/openapi/paths/agent_policies.yaml

```diff
@@ -34,6 +34,11 @@ get:
       in: query
       name: full
       description: When set to true, retrieve the related package policies for each agent policy.
+    - schema:
+        type: boolean
+      in: query
+      name: noAgentCount
```

Comment:
I would find it more intuitive to have a flag when we want to include `agentCount`, rather than having a flag to disable. Though this might be a bigger refactor if the API is already used in many places.

Reply:
I agree, I was so tempted to do it but this change is already larger than I expected. It would be a breaking change to the API